.github/workflows/ci.yml

```diff
@@ -340,15 +362,14 @@ jobs:
             :orange_book: Documentation preview is available from
             https://cognitedata.github.io/reveal-docs-preview/${{ env.PR_NUMBER }}/docs/next/.
 
-
   cleanup:
+    needs: [changes, ci-examples, build-viewer, deploy-preview-documentation]
+    if: ${{ needs.changes.outputs.should-run == 'true' && always() }}
```

Comment:
Is it necessary to keep the `always` part here? Or is it to ensure it runs even if the previous stages crash?